recommendations.md

```diff
@@ -50,14 +50,23 @@ MUST avoid forwarding these user credentials accidentally (see
 
 ### Authorization Code Grant {#ac}
 
-Clients MUST prevent injection (replay) of authorization codes into the
-authorization response by attackers. Public clients MUST use PKCE [@!RFC7636] to
-this end. For confidential clients, the use of PKCE [@!RFC7636] is RECOMMENDED.
-With additional precautions, described in (#nonce_as_injection_protection),
-confidential clients MAY use the OpenID Connect `nonce` parameter and the
-respective Claim in the ID Token [@!OpenID] instead. In any case, the PKCE
-challenge or OpenID Connect `nonce` MUST be transaction-specific and securely
-bound to the client and the user agent in which the transaction was started.
+Clients MUST prevent misuse of authorization codes and authorization code
```

Comment:
I would swap these because the "and" is kind of ambiguous this way.

> Clients MUST prevent authorization code injection attacks and misuse of authorization codes (see (#code_injection)) using...

Reply:
+1